ext/RMagick/rmimage.c

```diff
@@ -12,6 +12,20 @@
 
 #include "rmagick.h"
 
+#define BEGIN_CHANNEL_MASK(image, channels) \
+  { \
+    ChannelType \
+      channel_mask; \
+    channel_mask=SetPixelChannelMask(image, (ChannelType)channels)
```

Comment:
```
ChannelType channel_mask = SetPixelChannelMask(image, (ChannelType)channels)
```

Reply:
Updated the PR to use this kind of syntax.